ngraph/core/include/ngraph/type/element_type.hpp

```diff
@@ -37,7 +37,7 @@ namespace ngraph
 {
     namespace element
     {
-        enum class Type_t
+        enum Type_t
```

Comment:
Why we want to switch from `enum class` to `enum`?

Reply:
I tried to save the backward compatibility, in order for a code with `ngraph::element::i8` has a chance to continue work.